platform/ios/src/MGLMapView.mm

```diff
@@ -4629,6 +4622,15 @@ - (void)updateAnnotationViews
         else
         {
             annotationView.center = [self convertCoordinate:annotationContext.annotation.coordinate toPointToView:self];
+            UIView <MGLCalloutView> *calloutView = self.calloutViewForSelectedAnnotation;
+            if (calloutView && calloutView.representedObject == annotationContext.annotation) {
+                CGPoint point = annotationView.center;
+                point.y -= annotationView.bounds.size.height/2.0f;
```

Comment:
If we want it to look like MapKit did pre-iOS 10 (when it got rid of callouts of this style), then yes. But we don't have now / this PR still doesn't provide a facility for placement which moves the map before presentation the way SMCalloutView does (see [here](https://github.com/nfarina/calloutview/blob/d6ecaba377c9f963aef630faf86e3b8f8cdb88d1/SMCalloutView.h#L178-L190)).